### DISCUSSION TITLE
Blog Post: Deep Dive into Asset Interop

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#109] [General]
Hello!

I’m here with a deep dive into the last quest, Asset Interop. This quest is quite long so take your time to read and learn all of the Soroban concepts [here](https://dev.to/altug/soroban-quest-asset-interop-2jo4).

Hope you guys like it!

**Comment by achallagundla** [comment]
Heya,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM (a small token of gratitude for all your help with SQ)! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
